crates/type/src/tag/entry.rs

```diff
@@ -18,14 +18,14 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 #[serde(untagged)]
-pub enum Tag {
+pub enum Entry {
     Signed(Jws),
-    Unsigned(Entry),
+    Unsigned(tree::Entry),
 }
```

Comment:
Why rename `Tag` to `Entry`? Is this just the Tag's value/data, not the tag itself?

Reply:
Clippy is not happy about `Tag` residing within `tag` module.
In any case, `TagEntry` is a better name for something that is a signed or unsigned `TreeEntry`.
Unsigned tag entry is just a tree entry.